src/components/EmailList/EmailList.jsx

```diff
@@ -0,0 +1,63 @@
+import EmailItem from "../EmailItem/EmailItem";
+import SearchBar from "../SearchBar/SearchBar";
+import EmailString from "../EmailString"
+import { useEffect, useState } from "react"
+
+function EmailList() {
+
+    const [emailString, setEmailString] = useState(false)
+    const emailItems = [];
+
+    function fetchJSON() {
+        
+        useEffect(() => {
+            fetch("https://email-client-api.dev.io-academy.uk/emails")
+            .then(response => response.json())
+            .then(data => {
+                console.log(data)
+                setEmailString(data)
+                return(
+                    emailString
+                )
+            })
+        }, [])
+
+    }
+
+    fetchJSON();
+
+    const [filteredEmailItems, setFilteredEmailItems] = useState(emailString);
+
+    const handleSearch = (searchQuery) => {
+        const filteredItems = EmailString.filter(email => {
+            return email.name.toLowerCase().includes(searchQuery.toLowerCase()) ||
+                email.subject.toLowerCase().includes(searchQuery.toLowerCase()) ||
+                email.preview.toLowerCase().includes(searchQuery.toLowerCase()) ||
+                email.date_created.includes(searchQuery);
+        });
+        setFilteredEmailItems(filteredItems);
+        setEmailString(filteredItems);
+        console.log("working");
+    };
+
+    return (
+        <div>
+            <SearchBar handleSearch={handleSearch}/>
+        
+            <div className="overflow-scroll h-[720px] w-[400px]">
```

Comment:
Don't fight the tailwind styles with `[]`s unless you *really* have to. 

I don't know why you set height on this component. If the user has a tall screen they get a lot of whitespace at the bottom that could show them more emails. The height should be full or unset.

The width does need to be set to something, but the current value is too wide for *most* actual mobile screens, and the content is cut off to the right. You should use the [responsive breakpoints](https://tailwindcss.com/docs/responsive-design) and set appropriate widths. You can check against some presets using the Firefox dev tools responsive layout feature.